core/README.md

```diff
@@ -54,13 +54,13 @@ If it is not supported for your platform, you will see an error if you try to st
 
 ### Disabling `wandb-core`
 
-If you need to revert to the previous SDK backend while using `wandb < 0.18.0`, you can do so by following one of these steps:
+If you need to revert to the previous SDK backend while using `wandb >= 0.18.0`, you can do so by following one of these steps:
 
 **Option 1**: Modify your script Add the following line to your script:
```

Comment:
"Modify your script" should also be deleted

Reply:
yep!

Reply:
weird that i missed it lol